src/app/features/common-components/registration/registration.component.html

```diff
@@ -165,6 +165,27 @@ <h4 class="data team-name"> {{ signUpForm.get("teamName").value }} </h4>
             <p class="category">Email</p>
             <p class="data"> {{ signUpForm.get("schoolEmailAddress").value }} </p>
           </div>
+
+          <div class="info-group">
+            <p class="category">Phone Number</p>
+            <p class="data"> {{ signUpForm.get("phoneNumber").value }} </p>
+          </div>
+
+          <div class="info-group">
+            <p class="category">Developer Type</p>
+            <p class="data">{{ signUpForm.get("developerType").value?.viewValue}}</p>
+          </div>
+
+          <div class="info-group">
+            <p class="category"> Class </p>
+            <p class="data">{{ signUpForm.get("class").value?.viewValue}}</p>
+          </div>
+
+          <div class="info-group">
+            <p class="category"> Years of experience </p>
+            <p class="data">{{ signUpForm.get("prevParticipation").value.years}}</p>
```

Comment:
No numeral value is shown if the user doesn't select a value for years of experience. The default value should be 0. This could be implemented by including 0 has an option in the dropdown and changing its corresponding label/text to "How many years?" instead of "If yes, how many years?"